scripts/python/get_torus_fluxes.py

```diff
@@ -0,0 +1,159 @@
+#!/usr/bin/env python
+
+# © 2022. Triad National Security, LLC. All rights reserved.  This
+# program was produced under U.S. Government contract
+# 89233218CNA000001 for Los Alamos National Laboratory (LANL), which
+# is operated by Triad National Security, LLC for the U.S.  Department
+# of Energy/National Nuclear Security Administration. All rights in
+# the program are reserved by Triad National Security, LLC, and the
+# U.S. Department of Energy/National Nuclear Security
+# Administration. The Government is granted for itself and others
+# acting on its behalf a nonexclusive, paid-up, irrevocable worldwide
+# license in this material to reproduce, prepare derivative works,
+# distribute copies to the public, perform publicly and display
+# publicly, and to permit others to do so.
+
+import argparse
+import numpy as np
+import matplotlib
+import matplotlib.pyplot as plt
+import matplotlib.patches as patches
+from mpl_toolkits.axes_grid1 import make_axes_locatable
+import sys
+import shutil
+import os
+from subprocess import call, DEVNULL
+import glob
+import time
+from enum import Enum
+from phoebus_constants import cgs, scalefree
+import phoebus_utils
+from phoedf import phoedf
+
+def get_torus_fluxes(dfile):
```

Comment:
This is convenient. But why not just request phoebus to store and report them?

Reply:
Yeah I think once we have all the fluxes we care about in logfiles we can go ahead and delete this file from the repo